scripts/esbuild/utils/index.ts

```diff
@@ -16,20 +16,11 @@ export function getEsbuildAliases(): Record<string, string> {
     chalk: 'ansi-colors',
 
     // mapping aliases to top-level bundle entrypoints
-    '@stencil/core/testing': './testing/index.js',
-    '@stencil/core/compiler': './compiler/stencil.js',
-    '@stencil/core/dev-server': './dev-server/index.js',
-    '@stencil/core/mock-doc': './mock-doc/index.cjs',
-    '@stencil/core/internal/testing': './internal/testing/index.js',
-    '@stencil/core/cli': './cli/index.js',
-    '@sys-api-node': './sys/node/index.js',
-
-    // mapping node.js module names to `sys/node` "cache"
-    //
-    // these allow us to bundle and ship a dependency (like `glob`) as part
-    // of the Stencil distributable but also have our separate bundles
-    // reference the same file
-    glob: './sys/node/glob.js',
```

Comment:
What's the reason for dropping these? Just not needed in all instances of `getEsbuildAliases()` being called?

Reply:
Correct, some aliases are only needed in certain components e.g. `glob` which is why I removed it from here. The alias really depends on where the bundled file is located, e.g. how many levels they are nested within Stencil. Most artifacts are located within a directory, e.g. `/testing/index.js` or `/compiler/stencil.js` so I kept some global ones but made sure the packages that are located in a more deeper directory structure receive a custom alias.